packages/datatrak-web/src/views/SurveyPage.tsx

```diff
@@ -40,6 +48,10 @@ export const SurveyPage = () => {
     <SurveyContext>
       <Container>
         <Toolbar />
+        <TopProgressBar
+          currentSurveyQuestion={screenNumber}
+          totalNumberOfSurveyQuestions={numberOfScreens}
+        />
```

Comment:
If you add this to `components/SurveyScreen` instead of this file, you can use `numberOfScreens` from the `useSurveyForm` hook